test/integration/targets/pids/tasks/main.yml

```diff
@@ -0,0 +1,23 @@
+# Test code for the pid module
+# Copyright 2018, Saranya sridharan
+
+- name: "Installing PidOf"
+  yum:
+    name: sysvinit-tools
+    state: present
+  when: ansible_distribution == 'CentOS'
+
+- name: "Checking empty result" 
+  pids: 
+    name: "blahblah"
+  register: emptypids
+ 
+ 
+- name: "Verify the Process IDs (PIDs) returned is empty"
+  assert:
+    that:
+    - emptypids is not changed
+    - emptypids.pids == [] 
```

Comment:
Can you include a test that returns results as well? Ideally the test would cover cases of 0, 1 and 2 PIDs returned. Perhaps add a simple python script which sleeps for 10 seconds, run it, and use the module to find the PID(s).